manifests/params.pp

```diff
@@ -194,7 +194,7 @@
   # Zabbix-server
   $server_alertscriptspath                  = '/etc/zabbix/alertscripts'
   $server_allowroot                         = '0'
-  $server_cachesize                         = '8M'
+  $server_cachesize                         = '32M'
```

Comment:
this is technically a breaking change (as it changes a default), should it be split into an own PR that is labeled `breaking`?

Reply:
we could also alternatively configure it in the tests, but then users get a broken-by-default 6.0 install which just doesn't start with a 8M of cache